services/ir1/.hosts

```diff
@@ -0,0 +1,7 @@
+IPV4_PREFIX.61 ir01.LOCAL_DOMAIN
+IPV4_PREFIX.62 ir02.LOCAL_DOMAIN
+IPV4_PREFIX.63 ir03.LOCAL_DOMAIN
+IPV4_PREFIX.64 ir04.LOCAL_DOMAIN
+IPV4_PREFIX.65 ir05.LOCAL_DOMAIN
+IPV4_PREFIX.66 ir06.LOCAL_DOMAIN
+IPV4_PREFIX.67 ir07.LOCAL_DOMAIN
```

Comment:
why is it placed in `.../ir1/...` dir?

Reply:
Should I make a shared folder for such files? For example, just `.../ir/...`.

Reply:
yes, i would to that, otherwise it confuses